tools/block-generator/Makefile

```diff
@@ -1,41 +1,58 @@
-SCENARIO = scenarios/config.allmixed.small.yml
-SKIP = --skip-runner
+SCENARIO = scenarios/benchmarks/stress.50000.yml
+PG_UP = --start-postgres
 RESETDB = --reset-db
+TIMES = 1
 REPORTS = ../../tmp/RUN_RUNNER_OUTPUTS
 DURATION = 30s
 VERBOSE = --verbose
+CONDUIT = ./conduit
+TEMPLATE = # --template file-exporter (default postgres-exporter)
+PGUSER = algorand
+PGDB = generator_db
+PGCONT = "generator-test-container"
+PGCONN = "host=localhost user=$(PGUSER) password=algorand dbname=$(PGDB) port=15432 sslmode=disable"
+PGRUNNER = --postgres-connection-string $(PGCONN)
```

Comment:
A block comment near these would be helpful to describe that they are for tuning for the run-runner/pg-up/etc targets and typically useful for dev testing

Reply:
f29b623188d7c07e5929f4d8bd80dbbfcf196a55